opennms-doc/guide-admin/src/asciidoc/text/operation/timeseries/configuration.adoc

```diff
@@ -44,45 +44,53 @@ Additional configuration options are presented in the next section.
 
 The following properties, found in `${OPENNMS_HOME}/etc/opennms.properties`, can be used to configure and tune the _Timeseries Integration Layer_.
 
-[[ga-opennms-operation-newts-properties-general]]
+[[ga-opennms-operation-timeseries-properties-general]]
 ====== General
 [options="header, autowidth"]
 |===
 | Name                                            | Default              | Description
-| `org.opennms.timeseries.max_batch_size`       | `16`                 | Maximum number of records to insert in a single transaction. Limited by the size of the Cassandra cluster's batch_size_fail_threshold_in_kb property.
 | `org.opennms.timeseries.ring_buffer_size`     | `8192`               | Maximum number of records that can be held in the ring buffer. Must be a power of two.
-| `org.opennms.timeseries.config.writer_threads`       | `16`                 | Number of threads used to pull samples from the ring buffer and insert them into Newts.
+| `org.opennms.timeseries.config.writer_threads`       | `16`                 | Number of threads used to pull samples from the ring buffer and insert them into the Timeseries Database.
```

Comment:
```suggestion
| `org.opennms.timeseries.config.writer_threads`       | `16`                 | Number of threads used to pull samples from the ring buffer and insert them into the Time Series Database.
```

Reply:
@Bonrob2 do you suggest to use the two words 'Time Series' instead of the one word Timeseries? If so I think we should change it everywhere?

Reply:
Yes, two words in prose would be better, but I am aware that it is inconsistent with the rest of the documentation. This is something I'll update with the doc revamp for 27.0. If it's a lot of work right now, just leave it as one word, as long as it's one word consistently. I can note to update it in 27.0.

Reply:
I can update it in the time series integration layer and make it here consistent.